src/components/widgets/NumberWidget.js

```diff
@@ -120,12 +130,12 @@ export default class NumberWidget extends React.Component {
   render () {
     return (
       <input ref='input' className='number' type='text'
-        value={this.state.displayValue}
-        onKeyDown={this.onKeyDown}
-        onChange={this.onChange}
-        onMouseDown={this.onMouseDown}
-        onFocus={this.onFocus}
-        onBlur={this.onBlur}
+             value={this.state.displayValue}
```

Comment:
No. it's not a criteria you can consistently apply everywhere. Just use two spaces. You start to have context dependent indentation if you go that route. There's zero value on it.